aries-site/src/pages/templates/filtering.mdx

```diff
@@ -0,0 +1,111 @@
+
+If a user will likely be looking for a specific record as opposed to groups of data, consider implementing [search](/templates/searching).
+
+### Controls
+
+Filter controls should live above the data content and facilitate a user's ability to locate and take actions against records of interest.
+
+### Filters
+
+The way filters are displayed is dependent on the use case, number of filters, and data types of the content. These [general guidelines](#guidelines) are a starting point for how to present filtering in your application, but it's important to consider each recommendation in the context of your application's use cases.
+
+### Results
+
+Results should live below the controls and display any records that meet filter criteria. Depending on the data types and use case, results may be represented by a [List](#filtering-within-a-list), [Cards](#filtering-within-cards), or [Table](#filtering-within-a-table)
+
+### Pagination
+
+If the number of records displayed per page is limited, pagination should be placed underneath the results.
+
+## When to use
+
+### Filters in a center Layer
+
+When applying many filters at once, it is helpful to present them in a consolidated [Layer](/components/layer). In this example, the Layer is positioned at the center of the screen.
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/filtering/FilteringLists.js"
+  showResponsiveControls={['laptop', 'mobile']}
+  template
+>
+  <FilteringLists />
+</Example>
+
+### Filters in a right-anchored Layer
+
+When applying many filters at once, it is helpful to present them in a consolidated [Layer](/components/layer). In this example, the Layer is anchored to the right edge of the screen.
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/filtering/FilteringCards.js"
+  showResponsiveControls={['laptop', 'mobile']}
+  template
+>
+  <FilteringCards />
+</Example>
+
+## Guidelines
+
+Filtering should help users quickly narrow down data sets by selecting certain parameters to define the relevant data bounds. Different types of inputs may better suited for filtering depending on the type of data.
+
+Use your best judgement around which input type is appropriate for the data you are filtering. However, here are a few things to consider:
+
+- **Is data presented as Cards or List?** When there is ample screen space, display the filters to the left of the data as [a persistent column](/templates/filtering). Once space becomes limited, use a single filter button that opens a Layer containing the fitlers
+- **Is there more than one filter?** Use a single button with filter icon that opens a Layer containing the filters
+- **Is the user going to be selecting multiple options?** Filter with CheckBoxGroup
+- **Is the user aiming to filter based on a range?** Filter with RangeSelector
```

Comment:
Is there an intent to link to an example of this?

Reply:
Yes, adding that link in latest commit.